eppo_client/client.py

```diff
@@ -286,102 +288,112 @@ def get_bandit_action(
             },
             "default"
         )
-        if result.action is None:
-            do_variation(result.variation)
+        if result.action:
+            do_action(result.variation)
         else:
-            do_action(result.action)
+            do_status_quo()
         """
+        variation = default
+        action = None
         try:
-            return self.get_bandit_action_detail(
+            subject_attributes = convert_context_attributes_to_attributes(subject_context)
```

Comment:
`get_bandit_action_detail` was ensuring the attributes were passed in with context and then disaggregating them back down to `Attributes.`

Now that we pass these in before we try to ensure context, we want to do the opposite, which is to make sure they are disaggregated.